packages/datadog-plugin-redis/src/index.js

```diff
@@ -9,17 +9,20 @@ class RedisPlugin extends CachePlugin {
   static get system () { return 'redis' }
 
   start ({ db, command, args, connectionOptions = {}, connectionName }) {
-    if (!this.config.filter(command)) return this.skip()
+    const rawCommand = command
+    const normalizedCommand = command.toUpperCase()
+
+    if (!this.config.filter(normalizedCommand)) return this.skip()
 
     this.startSpan('redis.command', {
       service: getService(this.config, connectionName),
-      resource: command,
+      resource: rawCommand,
       type: 'redis',
       kind: 'client',
       meta: {
         'db.type': 'redis',
         'db.name': db || '0',
-        'redis.raw_command': formatCommand(command, args),
+        'redis.raw_command': formatCommand(normalizedCommand, args),
```

Comment:
Kind of confusing that the non-normalized value is called `rawCommand` and then _not_ used here where the key contains `raw_command`. Maybe `rawCommand` should be `resource` instead to use the short-hand object form and eliminate that minor confusion?

Reply:
I thought someone might point that out! It is kinda silly that the backend expects a field called `raw_command` which we've never passed a raw version of the command to. I've renamed `rawCommand` to `resource`.